### ISSUE TITLE
Fallback to online id_token validation 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#106]
```
If OpenSSL is not available fallback to verifying the id_token
using the tokeninfo endpoint:

  http://code.google.com/apis/accounts/docs/OAuth2Login.html#validatingtoken

```

Original issue reported on code.google.com by `jcgregorio@google.com` on 17 Feb 2012 at 12:54

**Comment by GoogleCodeExporter** [comment]
```
Currently on hold.
```

Original comment by `jcgregorio@google.com` on 22 Feb 2012 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
This may DoS the validation server, waiting on more feedback before proceeding.
```

Original comment by `jcgregorio@google.com` on 14 Jun 2012 at 7:47
- Changed state: **Waiting**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**
